{{cookiecutter.beat}}/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2016 {{cookiecutter.full_name}}
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
@ruflin I agree with you. Apache License 2 is a good default open-source license. It's not hard to change the LICENSE file afterwards.